.ansible-lint

```diff
@@ -0,0 +1,2 @@
+skip_list:
+  - '405'
```

Comment:
remove this and do it properly: https://devops.stackexchange.com/questions/5829/how-to-solve-e405-remote-package-tasks-should-have-a-retry-in-ansible